Apps/Src/main.c

```diff
@@ -139,16 +151,16 @@ void Task_Init(void *p_arg){
     );
     assertOSError(err);
 
-    // Initialize ReadTritium
+    // Initialize SendCarCAN
     OSTaskCreate(
-        (OS_TCB*)&ReadTritium_TCB,
-        (CPU_CHAR*)"ReadTritium",
-        (OS_TASK_PTR)Task_ReadTritium,
+        (OS_TCB*)&SendCarCAN_TCB,
+        (CPU_CHAR*)"SendCarCAN",
+        (OS_TASK_PTR)Task_SendCarCAN,
         (void*)NULL,
-        (OS_PRIO)TASK_READ_TRITIUM_PRIO,
-        (CPU_STK*)ReadTritium_Stk,
+        (OS_PRIO)TASK_SEND_CAR_CAN_PRIO,
```

Comment:
In Tasks.h, the PRIOs are not sequential because 7 is skipped. Is this something we want to fix so that it is incremental?

Reply:
Also, the last three (telemetry, debug dump) are only being used for tests. I know this PR isn't for Tasks.h, but does this need to be addressed?

Reply:
Will fix.